Sources/Shared/Core/VNError.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  VimeoNetworkingError.swift
```

Comment:
Do we have to keep the file name short due to linting?

Reply:
I renamed the class and forgot to rename the file, fixed!